astronomer/providers/amazon/aws/example_dags/example_batch.py

```diff
@@ -137,6 +139,25 @@ def get_job_queue_status() -> str:
         return "DELETED"
 
 
+def list_jobs_func() -> Any:
```

Comment:
job_is should be a string so what about returning `str` instead of `Any`?

Reply:
changed to `str`